integration/management-api-tests/src/test/java/org/wso2/micro/integrator/api/ConfigsResourceTestCase.java

```diff
@@ -0,0 +1,270 @@
+     * for logs.
+     *
+     * @throws IOException the io exception
+     */
+    @Test(groups = { "wso2.esb" }, dependsOnMethods = {"testRetrieveCorrelationConfigs"},
+            description = "Test get configs resource for correlation")
+    public void testUpdateCorrelationConfigs() throws IOException {
+
+        sendPutRequest(offset, true);
+        sendGetRequest(offset, true);
+
+        String apiLogFilePath = carbonHome + File.separator + "repository"
+                + File.separator + "logs" + File.separator + "correlation.log";
+        BufferedReader bufferedReader = new BufferedReader(new FileReader(apiLogFilePath));
+        String logLine = bufferedReader.readLine();
+        Assert.assertNotNull(logLine);
+        while ((logLine = bufferedReader.readLine()) != null) {
+            Assert.assertTrue(logLine.contains("HTTP State Transition") ||
+                    logLine.contains("ROUND-TRIP LATENCY") || logLine.contains("Thread switch latency"));
+        }
+    }
+
+    /**
+     * Test retrieve correlation configs with system parameter. This will start up the server
+     * with the enableCorrelationLogs flag. It will check whether the GET request of correlation
+     * config retrieves the proper status which is 'true'. It will check whether a PUT request
+     * will override the setting from the system property. ( Should not be able to override ).
+     * This is confirmed by reading the correlation log file again.
+     *
+     * @throws IOException                  the io exception
+     * @throws AutomationFrameworkException the automation framework exception
+     */
+    @Test(groups = { "wso2.esb" }, dependsOnMethods = {"testUpdateCorrelationConfigs"},
+            description = "Test get configs resource for correlation with system parameter")
+    public void testRetrieveCorrelationConfigsWithSystemParameter() throws IOException, AutomationFrameworkException{
+        stopServer();
+        startNewServer(true);
+        sendGetRequest(offset, true);
+        sendPutRequest(offset, false);
+
+
+        String apiLogFilePath = carbonHome + File.separator + "repository"
+                + File.separator + "logs" + File.separator + "correlation.log";
+        BufferedReader bufferedReader = new BufferedReader(new FileReader(apiLogFilePath));
+        String logLine = bufferedReader.readLine();
+        Assert.assertNotNull(logLine);
+        while ((logLine = bufferedReader.readLine()) != null) {
+            Assert.assertTrue(logLine.contains("HTTP State Transition") ||
+                    logLine.contains("ROUND-TRIP LATENCY") || logLine.contains("Thread switch latency"));
+        }
+
+
+        sendGetRequest(offset,true);
+
+        while ((logLine = bufferedReader.readLine()) != null) {
+            Assert.assertTrue(logLine.contains("HTTP State Transition") ||
+                    logLine.contains("ROUND-TRIP LATENCY") || logLine.contains("Thread switch latency"));
+        }
+
+
```

Comment:
Shall we remove these empty lines?

Reply:
Sure

Reply:
Fix the formatting issues and remove empty lines in this file